src/rpc/server.cpp

```diff
@@ -297,8 +297,17 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
             RPCHelpMan{"getrpcinfo",
                 "\nReturns details of the RPC server.\n",
                 {},
-                RPCResults{},
-                RPCExamples{""},
+                RPCResult{
+            "{ (json object with active commands)\n"
+            "  { (json object with information about an active command)\n"
```

Comment:
Should have the key `"active_commands":` saying it's a json array.

Reply:
fixed